src/components/Films/Films.tsx

```diff
@@ -44,6 +43,26 @@ const Films = () => {
     getFilmInfo();
   }, [title]);
 
+  const columns = [
```

Comment:
create separate file called  for exxample films-table.config.ts and move table config there, it means moving column definnitions